manifester/manifester.py

```diff
@@ -135,6 +136,7 @@ def verify_allocation_entitlements(self, entitlement_quantity, subscription_name
 
     def process_subscription_pools(self, subscription_pools, subscription_data):
         logger.debug(f"Finding a matching pool for {subscription_data['name']}.")
+        breakpoint()
```

Comment:
gotta remove these :) 

If you add pre-commit, there's a hook that will flag them.